hmy/api_backend.go

```diff
@@ -139,7 +139,16 @@ func (b *APIBackend) ProtocolVersion() int {
 // GetLogs ...
 func (b *APIBackend) GetLogs(ctx context.Context, blockHash common.Hash) ([][]*types.Log, error) {
 	// TODO(ricl): implement
-	return nil, nil
+	// return nil, nil
+	receipts := b.hmy.blockchain.GetReceiptsByHash(blockHash)
+	if receipts == nil {
+		return nil, nil
```

Comment:
should return error, not just nil

Reply:
updated in https://github.com/harmony-one/harmony/pull/2059/commits/ce78ec980f8ce096359e06bc8a2f31a71935a21c